crates/webidl/src/lib.rs

```diff
@@ -154,6 +195,10 @@ impl WebidlParse<()> for webidl::ast::Typedef {
 
 impl WebidlParse<()> for webidl::ast::NonPartialInterface {
     fn webidl_parse(&self, program: &mut backend::ast::Program, _: ()) -> Result<()> {
+        if is_chrome_only(&self.extended_attributes) {
```

Comment:
Would you add this to the other implemented `webidl::ast::*` that have extended attributes. Doing so will almost certainly require updating the `Event.rs` file under the test directory (which is good, since then your changes will actually be tested).

Reply:
So I added it to the other implemented `webidl::ast::*` and didn't have to make any changes to `Event.rs`, did I miss something?